instrumentation/apache-httpclient/apache-httpclient-5.2/library/README.md

```diff
@@ -0,0 +1,60 @@
+# Library Instrumentation for Apache Http client version 5.2
+
+Provides OpenTelemetry instrumentation for [Apache Http Client 5.2](https://hc.apache.org/httpcomponents-client-5.2.x/).
+
+## Quickstart
+
+### Add these dependencies to your project
+
+Replace `OPENTELEMETRY_VERSION` with the [latest
+release](https://search.maven.org/search?q=g:io.opentelemetry.instrumentation%20AND%20a:opentelemetry-apache-httpclient-5.2).
+
+For Maven, add to your `pom.xml` dependencies:
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-apache-httpclient-5.2</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+</dependencies>
+```
+
+For Gradle, add to your dependencies:
+
+```groovy
+implementation("io.opentelemetry.instrumentation:opentelemetry-apache-httpclient-5.2:OPENTELEMETRY_VERSION")
+```
+
+### Usage
+
+The instrumentation library provides a builder class `ApacheHttpClient5Telemetry` that wraps
```

Comment:
is `ApacheHttpClient5Telemetry` really a builder? does it really wrap the `HttpClientBuilder`?